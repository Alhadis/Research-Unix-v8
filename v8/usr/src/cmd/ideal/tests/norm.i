norm
.nf
.IS
...minx -1.5
...maxy 1.5
...maxx 1.5
...miny -1.5
...libfile hole
gridline {
	var a,b;
	var neg, pos;
	conn 0.5*(a+b) - neg to 0.5*(a+b) + pos;
}
main{
	conn (1,0) to (1,-1) using 10 gridline {neg = 1; pos = 0;} <a,b>;
	conn (0,-1) to (1,-1) using 10 gridline {neg = 0; pos = (0,1);} <a,b>;
	put hole{
		radius = 1;
		center = (0,0);
		opaque exterior;
	};
}
.IE
.IS
...forget gridline
.IE
