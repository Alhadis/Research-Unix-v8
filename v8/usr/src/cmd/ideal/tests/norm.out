norm
.nf
.IS
...minx -1.5
...maxy 1.5
...maxx 1.5
...miny -1.5
...maxx 0.998749
...maxy 0
...minx 0
...miny -0.998749
...line 0.95 0 0.95 -0.31225
...line 0.85 0 0.85 -0.526783
...line 0.75 0 0.75 -0.661438
...line 0.65 0 0.65 -0.759934
...line 0.55 0 0.55 -0.835165
...line 0.45 0 0.45 -0.893029
...line 0.35 0 0.35 -0.93675
...line 0.25 0 0.25 -0.968246
...line 0.15 0 0.15 -0.988686
...line 0.05 0 0.05 -0.998749
...line 0.31225 -0.95 0 -0.95
...line 0.526783 -0.85 0 -0.85
...line 0.661438 -0.75 0 -0.75
...line 0.759934 -0.65 0 -0.65
...line 0.835165 -0.55 0 -0.55
...line 0.893029 -0.45 0 -0.45
...line 0.93675 -0.35 0 -0.35
...line 0.968246 -0.25 0 -0.25
...line 0.988686 -0.15 0 -0.15
...line 0.998749 -0.05 0 -0.05
.IE
.IS
.IE
